lib/livebook/application.ex

```diff
@@ -293,7 +293,7 @@ defmodule Livebook.Application do
 
     cond do
       teams_key && auth ->
-        Application.put_env(:livebook, :teams_auth?, true)
+        Application.put_env(:livebook, :teams_auth, :online)
```

Comment:
We set this just to override it in the offline case. Instead, let's set this in each of the `"offline"`/`"online"` branches below.